templates/packer/ansible/roles/common/files/osbuild-composer-override.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
```

Comment:
This we can drop.

Reply:
Of course, I thought this was for worker but nah, ofc it wasn't.